lib/xml2.xml

```diff
@@ -22,15 +28,42 @@ such XML data files.</description>
     </implementation>
     <group>
       <requires interface="http://repo.roscidus.com/lib/zlib1g">
-      <environment insert="usr/lib" name="LD_LIBRARY_PATH"/>
-    </requires>
+        <environment insert="usr/lib" name="LD_LIBRARY_PATH"/>
+      </requires>
       <implementation id="sha1new=a7e5d754ed506f4e567fabeae05884d073a69433" released="2012-03-27" version="2.7.8">
-      <archive href="http://repo.roscidus.com/lib/xml2/libxml2-i386-2.7.8.dfsg-2+squeeze3.tar.bz2" size="809575"/>
-    </implementation>
+        <archive href="http://repo.roscidus.com/lib/xml2/libxml2-i386-2.7.8.dfsg-2+squeeze3.tar.bz2" size="809575"/>
+      </implementation>
       <implementation arch="Linux-x86_64" id="sha1new=9e9d62c9b0aec9ad55de90c0df16e08a8d29f373" released="2012-03-27" version="2.7.8">
-      <archive href="http://repo.roscidus.com/lib/xml2/libxml2-amd64-2.7.8.dfsg-2+squeeze3.tar.bz2" size="862772"/>
-    </implementation>
-    </group>
+        <archive href="http://repo.roscidus.com/lib/xml2/libxml2-amd64-2.7.8.dfsg-2+squeeze3.tar.bz2" size="862772"/>
+      </implementation>
+    </group>  
   </group>
-
-</interface>
\ No newline at end of file
+  <implementation arch="Windows-*" id="sha1new=865784de59ebfccf1b37d384fcfa7ec5c7b63dda" license="LGPL (GNU Lesser General Public License)" released="2001-12-23" version="2.4.12">
+    <command name="run" path="bin/xmllint.exe"/>
```

Comment:
Maybe `name="xmllint"` would be better hete.

The feed would no longer be directly runnable, but it would be more obvious to users what exactly is being executed if they are forced to type `0install run --command=xmllint http://repo.roscidus.com/lib/xml2`.

Reply:
FYI this means you cant look at the versions from the 0install GUI because it refuses to show that dialog without a run command

Reply:
Done
Note that this means that you cant open the versions dialog from the GUI since it checks for a run command

Reply:
Good point!

I've added a "Run specific version" checkbox to the "Run specific command" dialog box in 0install/0install-win@2e87182 to address this. I'll try to release this soon-ish.

Reply:
This causes 0install windows to think that there is a run command. 
On integration it creates a menu item and a command line alias for it.
its name is libxml2 and it fails on run because there is no run command.

Reply:
9b24fa8e fixes the issue with Zero Install for Windows suggesting a `libxml2` command-line alias.

The desktop integration logic still suggests a start menu entry for the `run` command though. Will look into that.